code-challenges.js

```diff
@@ -24,22 +24,63 @@ const fruit2 = "banana"
 
 // --------------------1) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
-const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
-const padres1998WorldSeriesRuns = [6, 3, 5, 3]
-// Expected output: 9
+// const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
+// const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+// // Expected output: 9
+
+const combined = padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns).length
+console.log(combined)
 
 // Pseudo code:
 
+// - first we have to combine the two arrays with the mutant concat
+// - then we add .lenght at the end to get the total of the two combined arrays
+// - we use console.log to give us the result of the lenght witch in this case is 9
+
 // --------------------2) Create the code that will reverse the letters of a string using the test variable provided below.
 
-const currentCohort = "Golf 2023"
+var currentCohort = ["Golf 2023"]
+
+var stringToArray = currentCohort.split()
+console.log(stringToArray)
+
+var stringToArray = currentCohort.split("")
+console.log(stringToArray)
+
+stringToArray.reverse()
+console.log(stringToArray)
+
 // Expected output: "3202 floG"
 
 // Pseudo code:
 
+// Reverse in javascript cant be applied to a string, but it can be applied to an array
+
+// - Var currentCohort selects the variable im selecting witch is Gold 2023
+
+// - i tried to convert the string into an area using the split method.
+
+// - i then tried to split up the string into every single character being on its own
+
+// - I then tried to reverse the array
+
+// - i could not get it to Worker, it kept telling me .split was not a function. 
```

Comment:
This is because you changed the variable from a string to an array. `.split` only works on strings, so when you tried to use it, it was being applied to an array, which caused an error. Don't change the given variables